docs/openapi/api-external-interop-v1.yaml

```
@@ -209,6 +209,137 @@ paths:
                 detail: 'There is a problem processing the request on the server.'
                 traceId: 'string'
                 errors: [{'code':'000-1234', 'detail':'The server is currently down for maintenance. Please try again later.'}]
+                
+   '/eservices/searchEservices':
+    get:
+      summary: Retrive e-services by filters
+      description: >-
+        Retrive e-services by filters : eserviceName , producerName, versionNumber, eservice-state 
```

These details are already in the `parameters` section (and we will likely forget to update this description when parameters will be updated)
```suggestion
      description: Retrive e-services by filters
```

---

done